.github/workflows/release.yml

```diff
@@ -20,3 +20,90 @@ jobs:
 
       - name: Push docker containers
         run: docker/push_all.sh
+
+  generate_docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Build Jazzer API docs
+        run: |
+          bazel build //deploy:jazzer-api-docs
+          cp bazel-bin/deploy/jazzer-api-docs.jar .
+
+      - name: Build Jazzer docs
+        run: |
+          bazel build //deploy:jazzer-docs
+          cp bazel-bin/src/main/java/com/code_intelligence/jazzer/jazzer-docs.jar .
```

Comment:
Instead of hard-coding the paths, we should use `bazel cquery --output=files`.

We could probably also merge these three separate builds into a single one with the three targets specified.

Reply:
> Instead of hard-coding the paths, we should use `bazel cquery --output=files`.

Yeah, that sounds sensible! Will change!

> We could probably also merge these three separate builds into a single one with the three targets specified.

While I have no strong opinions on this, I've split this up so that different steps would fail depending on which docs would fail to build. OTOH, you can still figure that out, if we unify them into one build step …